corpus_explorer/utils.py

```diff
@@ -198,3 +203,69 @@ def generate_topic_scatter_plot(topic_coordinates, topic_proportions):
             marker_size=topic_sizes,
         ),
     )
+
+
+def generate_visualization(topic_coordinates, topic_proportions):
+
+    x_coords = topic_coordinates[:, 0]
+    y_coords = topic_coordinates[:, 1]
+
+    # Scale proportion values to adequate Plotly marker size values
+    scaler = MinMaxScaler(feature_range=(MIN_MARKER_SIZE, MAX_MARKER_SIZE))
```

Comment:
why are these constants and not just passed in as 20 and 100 here. Are we anticipating changing this later?

Reply:
I preferred naming them, so that it's clear why we're scaling in that range.

Also related to whether to use constants or not:
I'm not entirely sure of how we'll want to handle resizing as we subset the corpus to be represented. For instance, should we rescale everything after subsetting? If so, then having the Min-Max range in a pair of constants is useful, because we'll be using them in more than one place.